test/reporters/base.spec.js

```diff
@@ -403,4 +403,20 @@ describe('Base reporter', function() {
     errOut = stdout.join('\n').trim();
     expect(errOut, 'to be', '1) test title:\n     Error\n  foo\n  bar');
   });
+
+  it('should let you stub out console.log without effecting reporters output', function() {
+    var logCached = console.log;
+    var logRes = [];
+
+    console.log = function(line) {
+      logRes.push(line);
+    };
+    var base = new Base({on: function() {}, stats: {duration: 0, passes: 3}});
```

Comment:
I think you can just use `Base.list(stuff)` instead of instantiating `Base` and calling `epilogue()`...

another (better?) way to test it would be to just use sinon; stub `console.log` then check that it was not called.

```js
// untested
sinon.stub(console, 'log');
Base.log('something');
expect(console.log, 'was not called');
```

Reply:
Nice, done.